### ISSUE TITLE
Access to the website in landscape orientation on a mobile device is difficult.

### ISSUE THREAD

**Issue by KingaWitkoAccessibility** [Problematy/goodmap#86]
**Website:** https://www.rzutberetem.eco/

**Description:**

Access to the website in landscape orientation on a mobile device is difficult. The logo and the top menu take up a large part of the screen and attempts to navigate and scroll the screen result in the map moving only.

**Expected result:**

1. The Top panel should not cover most of the screen.
2. There should be a mechanism to block the map from moving (e.g. use two fingers to move) or the map area should be limited so that there is a margin allowing additional navigation on the page.

**Attachment:**
![image](https://github.com/Problematy/goodmap/assets/173385806/321bbde1-80d3-4cd4-8e4b-6027d8cbc78a)

**Comment by x-Milosz** [comment]
1. Fixed with https://github.com/Problematy/platzky/pull/8
2. The map width and height is automatically adjusted., the left panel with check-boxes is scrollable. There is no more content after the map - there is no need to scroll.
